LondonBritishLibrary/orient/BLorient9798.xml

```diff
@@ -0,0 +1,281 @@
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="21"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Neat handwriting, possibly several hands.</desc>
+                                <date notBefore="1700" notAfter="1868" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus target="#56v"/>
+                                <desc>Notes in a Ḥasaba ḥǝgg, which notes the death of Emperor Tewodros and must be therefore in the year 
+                                    <date when-custom="521" calendar="grace" when="1868">521</date> or shortly after.</desc>
+                                <date when="1868">1868</date>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#49rb"/>
+                                    <desc type="GuestText">Calculation of the auspicious and ill-omened days 
+                                        (<ref type="work" corresp="#LIT4145Onausp">Ḥasāba ṣǝnǝʿt wa-ṣǝnṣǝnt</ref>).</desc>
+                                    <q xml:lang="gez"><title>ሐሳበ፡ ተአምር፡ ጽንዕት፡ ወጽንጽንት፡</title> በመስከረም፡ ፲ወ፪ጽንዕት። ወላዕሉ፡ ጽንፅንት። በጥቅምት፡ ፳፫ጽንፅንት፡ ወላዕሉ፡ ጽንዕት፡</q>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus target="#49v"/>
+                                    <desc type="GuestText">A fragment of an Amharic work concerning the Trinity.</desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote">In the blank space between the columns a note partly illegible with the name of an 
+                                        British army officer as owner of the manuscript.</desc>
+                                    <q xml:lang="en"><persName role="owner" ref="PRS14353Edwards">William Wilson Edwards</persName>, 
+                                        Capt. of Madras Light Cavalry and Abyssinian Transport Train 
+                                        <date notBefore="1867" notAfter="1868">1867-1868</date></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to Emperor 
```

Comment:
One , is not needed here, please use the numbers with the name of the king, for the sake of conviniency.